internal/proxy/task_insert.go

```diff
@@ -443,8 +447,11 @@ func (it *insertTask) Execute(ctx context.Context) error {
 	tr.Record("get used message stream")
 
 	channelNames, err := it.chMgr.getVChannels(collID)
+	traceID, _, _ := trace.InfoFromContext(ctx)
```

Comment:
log.Ctx(ctx).With(...), avoid logging traceID and collectionID again and again

Reply:
ok，I will do this in the whole package at the next pr.